openqa-monitor-investigation-candidates

```diff
@@ -7,26 +7,28 @@ failed_since="${failed_since:-"(timezone('UTC', now()) - interval '24 hour')"}"
 # use an environment variable "group_id" to limit the search to that job group, e.g.
 # "1" for "openSUSE Tumbleweed" on openqa.opensuse.org
 # shellcheck disable=SC2154
-group_query="${group_id+" group_id=$group_id and"}"
+group_query="${group_id:+" group_id=$group_id and"}"
 # use an environment variable "exclude_group" with an SQL regex string to
 # exclude all jobs that match the group name specified, e.g.
 # "%Not This Group%"
 exclude_group="${exclude_group:-"%Kernel%|%Development%|%Staging%|%MicroOS%"}"
-exclude_group_query="${exclude_group+" and job_groups.name not similar to '${exclude_group}'"}"
+exclude_group_query="${exclude_group:+" and job_groups.name not similar to '${exclude_group}'"}"
 # use an environment variable "exclude_parent" with an SQL regex string to
 # exclude all jobs that match the parent group name specified, e.g.
 # "%Development|Open Build Service%"
 exclude_parent="${exclude_parent:-"%Development|Open Build Service|Others%"}"
-exclude_parent_query="${exclude_parent+" where job_group_parents.name not similar to '${exclude_parent}'"}"
+exclude_parent_query="${exclude_parent:+" where job_group_parents.name not similar to '${exclude_parent}'"}"
 result="${result:-'failed'}"
 comment_query="${comment_query:-" and jobs.id not in (select job_id from comments where job_id is not null)"}"
+# optional additional query, e.g. "test ~ '%foo'"
+additional_query="${additional_query:+" and ${additional_query}"}"
```

Comment:
Can this be documented somewhere? How is this supposed to be used?

Reply:
of course all of this can be documented somewhere but this should be beyond the scope of this PR. Please let's focus on just adding the simple feature extension. You are welcome to provide help text, documentation, more tests, etc. but please outside the scope of this PR.

> How is this supposed to be used?

The comment line above shows how it can be used. The git commit message gives a complete example bash command line

Reply:
<img width="832" alt="image" src="https://github.com/os-autoinst/scripts/assets/1204189/70a84337-7d9b-4216-a024-434df57300b7">

I don't see any description here. If you provided at least an example, better also a ticket reference that would be totally fine by me.

Reply:
commit message != PR description